src/main/java/bio/terra/service/auth/iam/IamRole.java

```diff
@@ -6,6 +6,7 @@
 
 public enum IamRole {
   ADMIN,
+  MEMBER,
```

Comment:
No, but that’s intentional - this is a role specific to Firecloud groups, which have admins and members.

Reply:
Is there a good place to add more comments/color to this Role? It's a fairly generic name and I think it could be easily confused with the "user" role.

Reply:
That's good feedback!  I could pursue a slight reorganization akin to what we do for `IamAction`, trying to group by common roles then adding comments for resource-specific role groupings: https://github.com/DataBiosphere/jade-data-repo/blob/a9634d254c9e3caa3d7ce5f60408226b7dfb583b/src/main/java/bio/terra/service/auth/iam/IamAction.java

But this has the potential to go stale.  Minimally I can add a comment disambiguating. Thoughts?

Reply:
A comment sounds good. 
There isn't any wiggle room on the name of this role, is there?

Reply:
The name itself is not within our control, but we could expand the enum value definition to optionally allow some additional description or context to be specified.  I'll give that a shot.

Reply:
If that's too much, I'm good with just a comment.

Reply:
I ended up expanding the enum definition to allow for an optional embedded description.

Reply:
Oh, actually… I'm going to undo this!  A Javadoc-style comment is a better fit, and then you can see it in the IntelliJ pop-up window within your references.

Reply:
![Screen Shot 2022-11-23 at 12 44 29 PM](https://user-images.githubusercontent.com/79769153/203614507-e7de3177-e04b-4a50-8bfe-6718041e883d.png)